pom.xml

```diff
@@ -111,6 +116,13 @@
           <version>3.0.0</version>
           <scope>provided</scope>
         </dependency>
+        <!-- Fix test error due to fail to transform class AbstractProject when using javassist-3.18.1-GA -->
```

Comment:
From where does this Java Assist lib comes? If it comes from a plugin/core, it may indicate a real issue which may happen in the wild

Reply:
It is a test dependency. If you execute a dependency:tree, you can find:

[INFO] +- org.powermock:powermock-module-junit4:jar:1.6.1:test
[INFO] |  \- org.powermock:powermock-module-junit4-common:jar:1.6.1:test
[INFO] |     +- org.powermock:powermock-core:jar:1.6.1:test
[INFO] |     |  \- org.javassist:javassist:jar:3.18.2-GA:test

Using that version, next exception is thrown:
`java.lang.IllegalStateException: Failed to transform class with name hudson.model.AbstractProject. Reason: javassist.bytecode.InterfaceMethodrefInfo cannot be cast to javassist.bytecode.MethodrefInfo`

This error is fixed in javassist-3.19.0-GA, as you can see this issue: [JASSIST-220](https://issues.jboss.org/browse/JASSIST-220)